EIPS/eip-approval_by_amount.md

```diff
@@ -0,0 +1,122 @@
+---
+eip: to be assigned
+title: ERC-1155 Approval By Amount Extension
+description: Extension for ERC-1155 secure approvals
+author: Iván Mañús Murcia, Juan Carlos Cantó Martinez (@EscuelaCrypto in Telegram)
+discussions-to: https://ethereum-magicians.org/t/eip-erc1155-approval-by-amount/9898
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-11
+requires: 1155
```

Comment:
```suggestion
requires: 20, 1155
```

EIP-20 is referenced

Reply:
Name patterns are a require? I don't require ERC-20 for this extension